packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -266,21 +266,19 @@ export class WebviewWidget extends BaseWidget {
     waitReceiveMessage(object: WebviewWidget, resolve: any) {
         if (object.readyToReceiveMessage) {
             resolve(true);
+        } else {
+            setTimeout(this.waitReceiveMessage, 100, object, resolve);
```

Comment:
Don't we need to bind the function here?

Reply:
[Apparently not](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#As_an_object_method), my bad.